packages/superset-ui-chart-controls/src/types.ts

```diff
@@ -179,7 +181,7 @@ export type TabOverride = 'data' | 'customize' | boolean;
 export interface BaseControlConfig<
   T extends ControlType = ControlType,
   O extends SelectOption = SelectOption,
-  V = unknown
+  V = JsonValue
```

Comment:
It's a little complicated, but setting this allows `ControlState` to be used in [`controlUtils`](https://github.com/apache/superset/pull/13520/files#diff-e07e5c8190a73296c5a8192ac79a72cfe9eb1e93a076a4511b8879131c54a669) without a value incompatibility warning.